src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -1479,8 +1477,6 @@ public function isMethod($method)
      *
      * @see https://tools.ietf.org/html/rfc7231#section-4.2.1
      *
-     * @param bool $andCacheable Adds the additional condition that the method should be cacheable. True by default.
```

Comment:
This one should not be removed.

Reply:
I'll look on this one

Reply:
Oh, I see it's commented bellow. This Fixer is not able to differentiate between commented argument and extra useless docs for param.

Just revert this manually